guides/flutter-setup.md

```diff
@@ -54,60 +54,67 @@ Change the app's `AndroidManifest.xml` to include:
 
 Set the `com.apple.security.network.client` entitlement to `true` in the `macos/Runner/DebugProfile.entitlements` and `macos/Runner/Release.entitlements` files. You can also do this in Xcode by selecting the `macos` target, then the `Signing & Capabilities` tab, and checking `Outgoing connections (Client)` for both the Release and Debug targets of your app.
 
+## Verify Your Setup
 
-## Sending signals
+Build and run your app to verify that TelemetryDeck is properly integrated. The SDK automatically begins collecting data when initialized.
 
-Send a signal using the following method:
+{% notewarning "Test Mode for Development Signals" %}
+If your app's build configuration is set to "Debug", all signals sent will be marked as testing signals. You'll see them show up in the TelemetryDeck Dashboard when the **Test Mode** toggle under the tab bar is turned on. Remember to disable Test Mode in the dashboard to see your production data once your app is live.
+{% endnotewarning %}
+
+Open the TelemetryDeck Dashboard, navigate to "Explore > Recent Signals" and make sure "Test Mode" is enabled. You should see automatic signals appear after launching your app.
+
+---
+
+{% noteinfo "Ready for Basic Insights" %}
+Congratulations! With just the SDK integration you've completed, TelemetryDeck will automatically track user sessions, app launches, and device information. This basic setup provides valuable built-in insights without any additional code.
+
+You can now build and release your app. Once users start using it, your TelemetryDeck dashboard will begin showing data about user behavior, device types, and other key metrics.
+{% endnoteinfo %}
+
+## Enhancing Your Analytics (Optional)
+
+While basic session tracking provides valuable information, sending custom signals lets you answer questions specific to how users engage with *your* app.
+
+### Sending Custom Signals
+
+Send a simple signal using the following method:
 
 ```dart
 Telemetrydecksdk.send("signal_type")
 ```
 
-## Signals with additional attributes
-
-Append any number of custom attributes to a signal:
+You can append any number of custom attributes to a signal:
 
 ```dart
 Telemetrydecksdk.send("signal_type",
   additionalPayload: {"attributeName": "value"});
 }
 ```
 
-## Stop sending signals
-
-Prevent signals from being sent using the stop method:
+To temporarily stop sending signals (for example, if a user opts out):
 
 ```dart
 Telemetrydecksdk.stop()
 ```
 
 In order to restart sending events, you will need to call the `start` method again.
 
-## Test mode
+### Additional Configuration Options
```

Comment:
I don't think we need special wording for test mode for Flutter as it's the same behaviour as the KotlinSDK and SwiftSDKs. This section can be replaced with the same one from the other two pages.

Reply:
Hmm, I think you were referring to your comment above for Android, so I changed the section above (not the commented line) to:

```markdown
{% notewarning "When running from your IDE, you're sending test signals" %}
If your app is configured with a debug build type (i.e. the default build variant running from Android Studio), your signals will be tagged as **Test Signals**, meaning that you can easily filter them out later. You'll see them show up in the TelemetryDeck Dashboard when the **Test Mode** toggle under the tab bar is turned on.
{% endnotewarning %}
```